jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -51,13 +51,13 @@ async def applications_create(
     identity_claims = IdentityClaims.from_token_payload(token_payload)
     create_dict = dict(
         **application.dict(exclude_unset=True),
-        application_owner_email=identity_claims.email,
+        owner_email=identity_claims.email,
     )
 
     logger.debug("Inserting application")
 
     try:
-        insert_query = applications_table.insert().returning(applications_table)
+        insert_query = insert(JobScriptTemplate).returning(JobScriptTemplate)
         logger.trace(f"insert_query = {render_sql(insert_query)}")
         application_data = await database.fetch_one(query=insert_query, values=create_dict)
```

Comment:
I thought we would get rid of databases with this PR. Isn't it worthy?

Reply:
A code you referred to was a work in progress, the dependency `databases` was indeed removed from the project.